src/annotator/components/ClusterToolbar.tsx

```diff
@@ -68,7 +68,6 @@ function ClusterStyleControl({
               <div
                 style={{
                   backgroundColor: highlightStyles[styleName].color,
-                  textDecoration: highlightStyles[styleName].decoration,
```

Comment:
What is the story with text decorations in this PR? It looks like they are being removed as they are currently unused?